src/test/java/de/tum/in/www1/artemis/StudentScoreUtilService.java

```diff
@@ -0,0 +1,38 @@
+package de.tum.in.www1.artemis;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import de.tum.in.www1.artemis.domain.Exercise;
+import de.tum.in.www1.artemis.domain.User;
+import de.tum.in.www1.artemis.domain.scores.StudentScore;
+import de.tum.in.www1.artemis.repository.ResultRepository;
+import de.tum.in.www1.artemis.repository.StudentScoreRepository;
+
+/**
+ * Service responsible for initializing the database with specific testdata related to student scores for use in integration tests.
+ */
+@Service
+public class StudentScoreUtilService {
```

Comment:
can this be package private?

Reply:
All our util services are intended to be useable for all our integration testing needs. Therefore, all of them are public.